src/ImageRunModal.jsx

```diff
@@ -328,104 +331,124 @@ export class ImageRunModal extends React.Component {
-                <FormGroup fieldId='run-image-dialog-image' label={_("Image")} hasNoPaddingTop>
-                    <div id='run-image-dialog-image'> { image.RepoTags ? image.RepoTags[0] : "" } </div>
-                </FormGroup>
-
-                <FormGroup fieldId='run-image-dialog-name' label={_("Name")}>
-                    <TextInput id='run-image-dialog-name'
+                <Flex>
+                    <FlexItem align={{ default: 'alignLeft' }}>
+                        <FormGroup fieldId='run-image-dialog-name' label={_("Name")}>
+                            <TextInput id='run-image-dialog-name'
                                placeholder={_("Container name")}
                                value={dialogValues.containerName}
                                onChange={value => this.onValueChanged('containerName', value)} />
-                </FormGroup>
-
-                <FormGroup fieldId='run-image-dialog-command' label={_("Command")}>
-                    <TextInput id='run-image-dialog-command'
-                               placeholder={_("Command")}
-                               value={dialogValues.command || ''}
-                               onChange={value => this.onValueChanged('command', value)} />
-                </FormGroup>
-
-                <FormGroup fieldId='run-image-dialog-memory' label={_("Memory limit")}>
-                    <InputGroup className="ct-input-group-spacer-sm" id="run-image-dialog-memory-limit">
-                        <Checkbox id="run-image-dialog-memory-limit-checkbox"
+                        </FormGroup>
+                    </FlexItem>
+                </Flex>
+                <Tabs activeKey={activeTabKey} onSelect={this.handleTabClick}>
+                    <Tab eventKey={0} title={<TabTitleText>Details</TabTitleText>} className="pf-l-grid pf-m-gutter">
+                        <FormGroup fieldId='run-image-dialog-image' label={_("Image")} hasNoPaddingTop>
+                            <div id='run-image-dialog-image'> { image.RepoTags ? image.RepoTags[0] : "" } </div>
+                        </FormGroup>
+
+                        <FormGroup fieldId='run-image-dialog-command' label={_("Command")}>
+                            <TextInput id='run-image-dialog-command'
+                           placeholder={_("Command")}
+                           value={dialogValues.command || ''}
+                           onChange={value => this.onValueChanged('command', value)} />
+                        </FormGroup>
+
+                        <FormGroup fieldId="run=image-dialog-tty">
+                            <Checkbox id="run-image-dialog-tty"
+                              isChecked={this.state.hasTTY}
+                              label={_("With terminal")}
+                              onChange={checked => this.onValueChanged('hasTTY', checked)} />
+                        </FormGroup>
+
+                        <FormGroup fieldId='run-image-dialog-memory' label={_("Memory limit")}>
```

Comment:
fieldId needs to match nested field Id. The reason is that it uses it for the aria-labelled-by accesibility property.

Reply:
In this case you can use the input group's id.

Reply:
Fixed everywhere it applies.